src/WebJobs.Script/Description/Proxy/ProxyFunctionInvoker.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.AppService.Proxy.Client.Contract;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal class ProxyFunctionInvoker : FunctionInvokerBase
+    {
+        private IProxyClient _proxyClient;
+
+        public ProxyFunctionInvoker(ScriptHost host, FunctionMetadata functionMetadata, IProxyClient proxyClient) : base(host, functionMetadata)
+        {
+            _proxyClient = proxyClient;
+        }
+
+        protected override async Task InvokeCore(object[] parameters, FunctionInvocationContext context)
+        {
+            await _proxyClient.CallAsync(parameters, null, context.Logger);
```

Comment:
I notice this function doesn't take in the proxy name. Does this mean we're doing routing twice now? Once for Script webhost to find the FunctionMetdata --> FunctionInvoker -->ProxyInvoker; and then the proxy gets the request and applies routing again?

---
In reply to: [132558688](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r132558688) [](ancestors = 132558688)